### ISSUE TITLE
composition over inheritence

### ISSUE THREAD

**Issue by mstrYoda** [dwmkerr/hacker-laws#112]
It might be add as a princibple.

**Comment by lusocoding** [comment]
I agree... it should be added. Link to Wikipedia here: [](https://en.wikipedia.org/wiki/Composition_over_inheritance)
